plugin/main/src/kotlinx/benchmark/gradle/JsMultiplatformTasks.kt

```diff
@@ -34,7 +34,7 @@ private fun Project.createJsBenchmarkCompileTask(target: JsBenchmarkTarget): Kot
         sourceSet.kotlin.setSrcDirs(files("$benchmarkBuildDir/sources"))
         sourceSet.resources.setSrcDirs(files())
         sourceSet.dependencies {
-            implementation(compilation.compileDependencyFiles)
+            implementation(files(provider { compilation.compileDependencyFiles }))
```

Comment:
Is this the recommended substitution? Does this ensure that `compileDependencyFiles` configuration is resolved when `implementation` is resolved?

Reply:
I am not sure if it is the recommended or an equivalent substitution. It is what I found in other projects that add a configuration as a source set dependency. Review from the kotlin-gradle-plugin team should help to answer those questions.

Reply:
Ok, please show it to someone from the team then.

Reply:
I would propose to change here to:
```kotlin
configurations[sourceSet.implementationConfigurationName].extendsFrom(compilation.compileDependencyConfigurationName)
```

Reply:
Done